docs/network/interfaces_and_networks.md

```diff
@@ -165,25 +169,30 @@ spec:
     }'
 ```
 
-Define a VMI with a [Multus](https://github.com/intel/multus-cni)
+Define a VMI with a [Multus](https://github.com/k8snetworkplumbingwg/multus-cni)
```

Comment:
Please consider dropping the link to the Multus repo, as it exists prior to this point.

Reply:
done